_docs/_latest/setup/real-time-enforcer.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Deploy Forseti Real Time Enforcer on GCP
+order: 005
+---
+
+# {{ page.title }}
+
+> _This is a beta release of Forseti Real Time Enforcer. This product might be changed in backward-incompatible ways 
```

Comment:
nit: since we are re-using this so often, can this be refactored out and pulled in via `include`?

Reply:
Updated.